tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -10,237 +10,139 @@ Feature: accept/decline shares coming from internal users
-    And user "user2" has shared folder "/simple-folder" with group "grp1"
-    And user "user1" has logged in using the webUI
-    When the user declines share "simple-folder" offered by user "User Two" using the webUI
-    And the user reloads the current page of the webUI
-    Then folder "simple-folder" should be in state "Declined" in the shared-with-you page on the webUI
-    And folder "simple-folder (2)" should not be listed in the files page on the webUI
+    Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
+    And these groups have been created:
+      | groupname |
+      | grp1      |
+    And user "user2" has been added to group "grp1"
+    And user "user1" has shared folder "/simple-folder" with user "user2"
+    And user "user1" has shared folder "/simple-folder" with group "grp1"
+    And the user has browsed to the shared-with-me page
+    When the user declines share "simple-folder" offered by user "User One" using the webUI
+    Then folder "simple-folder" shared by "User One" should be in "Declined" state on the webUI
+    When the user browses to the files page
+    Then folder "simple-folder (2)" should not be listed on the webUI
 
-  @skip @yetToImplement
   Scenario: reshare a share that you received to a group that you are member of
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been disabled
-    And user "user2" has shared folder "/simple-folder" with user "user1"
-    And user "user1" has logged in using the webUI
-    When the user accepts share "simple-folder" offered by user "User Two" using the webUI
+    Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
+    And these groups have been created:
+      | groupname |
+      | grp1      |
+    And user "user2" has been added to group "grp1"
+    And user "user1" has shared folder "/simple-folder" with user "user2"
+    And user "user2" has accepted the share "simple-folder" offered by user "user1"
     And the user has browsed to the files page
-    And the user shares folder "simple-folder (2)" with group "grp1" using the webUI
-    And the user declines share "simple-folder (2)" offered by user "User Two" using the webUI
-    And the user reloads the current page of the webUI
-    Then folder "simple-folder (2)" should be in state "Declined" in the shared-with-you page on the webUI
-    And folder "simple-folder (2)" should not be listed in the files page on the webUI
+    When the user shares folder "simple-folder (2)" with group "grp1" as "Viewer" using the webUI
+    And the user unshares folder "simple-folder (2)" using the webUI
+    And the user browses to the shared-with-me page using the webUI
+    Then folder "simple-folder" shared by "User One" should be in "Declined" state on the webUI
+    And folder "simple-folder" shared by "User Two" should be in "Pending" state on the webUI
```

Comment:
that share should not be listed here. I've opened an issue #2512
please mark the test with this issue